docs/openhabian.md

```diff
@@ -67,7 +67,7 @@ The included **openHABian Configuration Tool** [`openhabian-config`](#openhabian
 
 ### Hardware recommendation
 
-Let's put this first: our current recommendation is to get a RPi 4 with 2 or 4 GB of RAM, a 3A power supply and a 16 GB SD card.
+Let's put this first: our current recommendation is to get a RPi 4 with 2 or 4 GB of RAM, a 3A power supply and a 16 GB SD card. Because of the higher limit of write cycles, an SD card designed for Dashcams is preferable. Most manufacturers name them with a postfix like "Endurance" (e.g. "SanDisk MAX ENDURANCE" or "Kingston High Endurance"). 
```

Comment:
Good idea, but for code reviewing purposes please put each sentence on a new line.